turbo/jsonrpc/eth_call.go

```diff
@@ -502,11 +502,15 @@ func (api *APIImpl) CreateAccessList(ctx context.Context, args ethapi2.CallArgs,
 
 	// Retrieve the precompiles since they don't need to be added to the access list
 	precompiles := vm.ActivePrecompiles(chainConfig.Rules(blockNumber, header.Time))
+	excl := make(map[libcommon.Address]struct{})
```

Comment:
From & to should also be added to `excl`

Reply:
Is there a specific reason this should be handled both during tracing and after tracing? As I understand it, the resulting access list would be the same. 
The way it is handled now `excl` also has very clear semantics of "this address should under no circumstances appear in the access list", which would make it easy to use with user-provided addresses as well. Whereas using `excl` for to and from would require differentiating between `addAddress` and `addSlot` again.

Reply:
I was worrying about preserving backwards compatibility in the non-optimized case, but perhaps it's not that important. Now I'm thinking about making optimizeGas the default unless set to false explicitly.